app/lint-baseline.xml

```diff
@@ -0,0 +1,15 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<issues format="6" by="lint 8.9.0" type="baseline" client="gradle" dependencies="false" name="AGP (8.9.0)" variant="all" version="8.9.0">
+
+    <issue
+        id="AppLinkUrlError"
+        message="Android App Links&apos; `host` attributes must be valid web domains"
+        errorLine1="                    android:host=&quot;localhost&quot;"
+        errorLine2="                                  ~~~~~~~~~">
```

Comment:
This spacing looks a bit weird. Checking this is what we want.

Reply:
This is auto generated so that this lint will not throw errors, so it is how it is.